app/views/welcome/index.html.erb

```diff
@@ -1,21 +1,19 @@
-<main class="welcome" role="main">
```

Comment:
It looks like there's CSS tied to this and no other uses of it.  Could you check if we can get rid of it?